### DISCUSSION TITLE
Nested CRUD of hasMany relationship on edit page

### DISCUSSION THREAD

**Discussion by delfinn55** [Laravel-Backpack/community-forum#819] [Q&A (Help)]
Hello! I have two related models: Customer and Bid ($customer->bids(): hasMany).

Need to display a table with his bids on the user’s edit page. I used the package https://github.com/onkbear/backpack-nested-crud for this and set standard list view for nested CRUD:
```php
CRUD::setListView('nested_crud::list');
```

 But it loaded full theme layout [```/admin/customer/{id}/edit#bids``` route]:

![изображение](https://github.com/Laravel-Backpack/community-forum/assets/21172796/d7054a95-eb8c-456d-8ec6-f8ae50076618)

What is the best way to include the template to make it work (with dataTables, search, pagination, filters and other CRUD list possibilities)?

```
### PHP VERSION:
PHP 8.2.13 (cli) (built: Nov 21 2023 09:55:59) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.13, Copyright (c) Zend Technologies
    with Xdebug v3.2.1, Copyright (c) 2002-2023, by Derick Rethans
    with Zend OPcache v8.2.13, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
10.25.1.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.2.1
backpack/crud: 6.2.1
backpack/devtools: 2.0.3
backpack/filemanager: 3.0.4
backpack/generators: v4.0.2
backpack/langfilemanager: 5.0.0
backpack/logmanager: v5.0.1
backpack/menucrud: v4.0.1
backpack/pagemanager: 3.2.0
backpack/permissionmanager: 7.0.1
backpack/pro: 2.0.13
backpack/settings: 3.1.0
backpack/theme-tabler: 1.1.1

```

**Comment by karandatwani92** [comment]
Hey @delfinn55 

A functional nested CRUD with data tables, search, pagination, and filters is not possible right now.
This feature has already grabbed our attention and is in our TODOs.

For now, you may try https://github.com/izica/relations-widgets-for-backpack. This lists related items as a widget.
We haven't tried this package, so not sure how much difference it can make.

Let us know if this helps.
